runtime/gc_glue_java/MixedObjectScanner.hpp

```diff
@@ -53,11 +53,13 @@ class GC_MixedObjectScanner : public GC_HeadlessMixedObjectScanner
 protected:
 	/**
 	 * @param env The scanning thread environment
-	 * @param[in] objectPtr the object to be processed
-	 * @param[in] flags Scanning context flags
+	 * @param objectPtr the object to be processed
+	 * @param scanPtr
+	 * @param size
+	 * @param flags Scanning context flags
 	 */
-	MMINLINE GC_MixedObjectScanner(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, uintptr_t flags)
-		: GC_HeadlessMixedObjectScanner(env, J9GC_J9OBJECT_CLAZZ(objectPtr, env), env->getExtensions()->mixedObjectModel.getHeadlessObject(objectPtr), flags)
+	MMINLINE GC_MixedObjectScanner(MM_EnvironmentBase *env, fomrobject_t *scanPtr, uintptr_t size, uintptr_t flags)
```

Comment:
I am not sure change in `GC_HeadlessMixedObjectScanner` constructor is necessary. You can get `*scanPtr` and `size` from objectPtr easily (would look a little bit bulky)